docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
Could we move those build files to another folder than `docs/`? I'm afraid that they will be served via Github Pages if we let then here (ok I think that we could find a way to exclude them but it would be clearer to have in another location, like `.sphinx/`). WDYT?